src/middle/Fun_kind.ml

```diff
@@ -31,10 +31,17 @@ let suffix_from_name fname =
   then FnLpdf false
   else FnPlain
 
-let pp pp_expr ppf = function
+let pp pp_expr ppf kind =
+  let with_unnormalized_suffix (name : string) =
+    Option.merge
+      ~f:(fun x _ -> x)
+      ( String.chop_suffix ~suffix:"_lpdf" name
+      |> Option.map ~f:(fun n -> n ^ "_lupdf") )
+      ( String.chop_suffix ~suffix:"_lpmf" name
+      |> Option.map ~f:(fun n -> n ^ "_lupmf") ) in
```

Comment:
It is a copy to avoid a dependency cycle:
```
Error: Dependency cycle between:       
   _build/default/src/middle/.middle.objs/middle__Fun_kind.impl.all-deps
-> _build/default/src/middle/.middle.objs/middle__Expr.intf.all-deps
-> _build/default/src/middle/.middle.objs/middle__SizedType.impl.all-deps
-> _build/default/src/middle/.middle.objs/middle__Type.impl.all-deps
-> _build/default/src/middle/.middle.objs/middle__Utils.impl.all-deps
-> _build/default/src/middle/.middle.objs/middle__Fun_kind.impl.all-deps
```

Avoiding it seems more painful than the duplication in this case

Reply:
D'oh! In that case just make it public and remove `Utils.with_unnormalized_suffix`.

Reply:
Done